src/Console/Shell.php

```diff
@@ -307,6 +310,7 @@ public function dispatchShell()
         if (is_string($args[0]) && count($args) === 1) {
             $args = explode(' ', $args[0]);
         }
+        $args[] = '--requested';
```

Comment:
Something more tied to the root ``Shell`` class ?
Like the ``bare`` parameter used to prevent layouts from being rendered when a request is dispatched with ``requestAction()`` ?

I can take a look in that direction and see what I can come up with.